lms/resources/_js_config/__init__.py

```diff
@@ -102,18 +102,19 @@ def asdict(self):
         """
         return self._config
 
-    def enable_canvas_oauth2_redirect_error_mode(
+    def enable_oauth2_redirect_error_mode(
         self,
         error_details,
         auth_url=None,
         is_scope_invalid=False,
-        requested_scopes=None,
+        canvas_scopes=None,
```

Comment:
Scopes are a general OAuth2 thing but I think as far as this code goes "Canvas scopes" and "Blackboard scopes" might be two different things: the presence of `canvas_scopes` here triggers some very Canvas-specific text explaining that your "developer key" (a Canvas term) is missing required scopes and linking to Canvas-specific docs for fixing it. If we had Blackboard scopes or any other kind of scopes it's unlikely we'd want to trigger the same text. (As it happens we don't yet have any other types of OAuth scopes anyway, only Canvas ones.)